testsuite/ui/widgets/__init__.py

```diff
@@ -336,6 +336,57 @@ def get_response_code(self):
         return self.browser.text(self.RESPONSE_CODE_LOCATOR, parent=self)
 
 
+# pylint: disable=abstract-method
+# Widget contains fill method which raise not implemented exception if widget is not fillable but pylint detect it as
+# an abstract method
+class ActiveDocV3Section(Widget):
+    """Active Doc V3 preview section"""
+    ROOT = ParametrizedLocator('//*[@class="operation-tag-content"]')
+    ENDPOINTS_LIST = './span/div'
+    ITEMS_LOCATOR = './span/div/div/button/span[2]/a/span'
+    ITEMS_EXPAND_LOCATOR = './span/div/div/button/span[text()="{}"]/../span[2]/a/span[contains(text(),"{}")]'
+    RESPONSE_CODE_LOCATOR = './/*[text()="Server response"]/../table/tbody/tr/td[1]'
+    TRY_OUT_BUTTON_LOCATOR = '//*[@class="btn try-out__btn"]'
+    SELECT_OPTION_LOCATOR = '//select/option[text()="{}"]'
+    EXECUTE_BUTTON_LOCATOR = '//*[@class="btn execute opblock-control__btn"]'
+
+    def __init__(self, parent=None, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+
+    @property
+    def endpoints(self):
```

Comment:
this method is defined as abstract in `ParametrizedView`

Reply:
After discussion with @dhlavac  we agreed that these changes (this one and next two) are unnecessary